tests/Behat/CustomerContext/Builder/CustomerTestDataBuilder.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Behat\CustomerContext\Builder;
+
+use App\Core\Customer\Domain\Entity\Customer;
+use App\Core\Customer\Domain\Entity\CustomerStatus;
+use App\Core\Customer\Domain\Entity\CustomerType;
+use App\Core\Customer\Domain\Factory\CustomerFactoryInterface;
+use App\Shared\Domain\ValueObject\Ulid;
+use Faker\Generator;
+
+final class CustomerTestDataBuilder
+{
+    private string $initials;
```

Comment:
It will be better to declare all class fields inside the constructor instead of using this old approach

Reply:
If we use this approach in CustomerTestDataBuilder.php, we must add it to phpmd.tests:
```
        <exclude name="BooleanArgumentFlag"/>
        <exclude name="ExcessiveParameterList"/> 
```